### ISSUE TITLE
Does gl4es define default shaders?

### ISSUE THREAD

**Issue by oomek** [ptitSeb/gl4es#89]
When an opengl draw call has no shader specified does gl4es generate a default shader for it?

**Comment by ptitSeb** [comment]
Yes it does (if it's using GLES2 backend, wich is the case by default).

Note that it seems that on RPi, you need to allocate 128MB to graphic card (or something like that), or you may have various issue on rendering.
